### ISSUE TITLE
AttributeError: CTkToolTip object has no attribute 'block_update_dimensions_event'

### ISSUE THREAD

**Issue by LaserLisa** [Akascape/CTkToolTip#20]
I have an reproducable error occuring when moving my ctk window on another screen with a different resolution. 
This happens as soon as I hover over the widget with the tooltip (when the app has moved to the 2nd screen).
![image](https://github.com/Akascape/CTkToolTip/assets/126000198/974381fa-031d-456a-a1e7-caf370e3b8f3)
I can reproduce it with the example code given in the README.
It seems that tkinter has an automatic update dimensions event if the resolution of the screen is changing which is not defined for the ToolTip
I started the App on 1920x 1080 screen and moved it to 2560 x 1440.

I am using Windows 11 and CTkToolTip v0.8

**Comment by Phystek** [comment]
I have this issue too. 
When i move my window to another monitor, one with a different resolution, the tooltips stop working properly and gives the error shown by LaserLisa -AttributeError: 'CTkToolTip' object has no attribute 'block_update_dimensions_event'
This also then interferes with the canvas resizing when I go back to the original screen.
Is there any fix/workaround for this?

**Comment by PabloRiveroFOSSA** [comment]
if you just want to fixt it, even if its not a good fix. Adding this code to the class CTkToolTip will work:

```
def block_update_dimensions_event(self) -> True:
        return True

def unblock_update_dimensions_event(self) -> True:
        return True
```

**Comment by Phystek** [comment]
A further comment on this. 
In my case, I've found that it is only triggered when one display has scaling set (e.g. 150% in  Windows display settings - Scale) and the other doesn't (e.g. just a 100% in Windows display settings - Scale). Do perhaps not an issue with two displays being of different resolution, but rather with different scale settings in Windows.

**Comment by PabloRiveroFOSSA** [comment]
Yeah I'm pretty sure that's the case, I had quite a lot of issues with CTK when dealing with several screen at different scalings. I ended up dity fixing it by dividing the screen size by the screen scaling depending on the screen the program appears on. And I'm pretty sure this comes from the same root problem

**Comment by kabyru** [comment]
Hello @Akascape -- I have submitted https://github.com/Akascape/CTkToolTip/pull/37 to resolve this issue. Please take a look and let me know what you think. Thank you!

**Comment by kabyru** [comment]
Hi @Akascape -- checking in, can we take a look at this issue soon? Thanks!

**Comment by kabyru** [comment]
Hello @Akascape -- checking in, can we look at this issue? I believe that @saschabuehrle and I have created duplicate PRs to address this problem. Reach out if you have questions. Thanks.

**Comment by ajkessel** [comment]
> Hello [@Akascape](https://github.com/Akascape) -- checking in, can we look at this issue? I believe that [@saschabuehrle](https://github.com/saschabuehrle) and I have created duplicate PRs to address this problem. Reach out if you have questions. Thanks.

FWIW, [Custom2kinter](https://github.com/FedericoSpada/Custom2kinter) has implemented a CTkTooltip that does not have this bug.
